lib/spaceship/tunes/tunes_client.rb

```diff
@@ -314,10 +314,10 @@ def app_version(app_id, is_live)
 
       # We only support platforms that exist ATM
       platform = platforms.find do |p|
-        ['ios', 'osx'].include? p['platformString']
+        ['ios', 'osx', 'appletvos'].include? p['platformString']
```

Comment:
Yes that's true if the platforms array changes order at all we could do the wrong thing. I probably just got lucky with my testing. Let me think about this a bit more.